setuptools/tests/test_wheel.py

```diff
@@ -612,8 +612,17 @@ def sys_tags():
         for t in parse_tag('cp36-cp36m-manylinux1_x86_64'):
             yield t
     monkeypatch.setattr('setuptools.wheel.sys_tags', sys_tags)
-    assert Wheel(
-        'onnxruntime-0.1.2-cp36-cp36m-manylinux1_x86_64.whl').is_compatible()
+    # Clear the supported tags cache, otherwise the sys_tags monkeypatch
+    # has no effect.
+    setuptools.wheel._supported_tags.cache_clear()
```

Comment:
The errors in the CI are probably related to the fact that the function is `_get_supported_tags` instead of `_supported_tags`.

Reply:
Strange, I saw the tests passing locally with my code.  Well, there were a few failures, but they were there on the main branch as well.  Maybe I missed something in the big test output.
Your changes are good.  Thanks!